### PR TITLE
Updates resume module to use new resume guid and templates.

### PR DESCRIPTION
**Pull request by chrisbay**

### CODE REVIEW

**Comment by chrisbay** [comment]
[Change Resume Assignment so that it matches the Candidate Resume Template](https://trello.com/c/fVl8LXE6/205-change-resume-assignment-so-that-it-matches-the-candidate-resume-template)

File: src/modules/career/online-profiles.rst [hunk 1]
```diff
@@ -14,211 +14,68 @@ things to include, as well as things to leave out of each profile.
...
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For the Apprenticeship Program, we require you to complete a resume using `our template guide <https://docs.google.com/document/d/1Xr0O_ZnNizlWlUsxxCUZHL483NZWV7mmTCbWUhNVm74/view>`_. We require resumes of this format in order to enter our Apprenticeship program. Companies appreciate the shared template and it makes it easier for you to focus on the content of your resume rather than the style
 
-After clearly marking your personal information at the top of the page,
-the next section should briefly outline your technical skills. You should
-prioritize programming languages, frameworks, and major libraries over
-operating systems and software. You should order your skills from most
-familiar to least familiar. As a final note, you should also consider the
-type of job you are looking for.
+Upon clicking the link above you will be brought to this page.
```
Comment (CarlyLanglois) [inline src/modules/career/online-profiles.rst]: add a comma after above

File: src/modules/career/online-profiles.rst [hunk 2]
```diff
@@ -14,211 +14,68 @@ things to include, as well as things to leave out of each profile.
...
-
-.. tip:: 
+   LaunchCode Resume Guide
 
-   You should not include every language or technology you’ve heard of. You
-   should only include skills you would feel comfortable using to build a
-   project today. You’ve learned the skill and have some practical
-   experience with the skill. This will usually mean an entry level
-   developer will include fewer than 10 skills on their resume.
+Read through the entire guide. At the bottom you will links to four different templates for individuals with varying backgrounds. Choose the option that best describes you.
```
Comment (CarlyLanglois) [inline src/modules/career/online-profiles.rst]: At the bottom, you will find links to four different templates. Choose the option that best describes your background.

File: src/modules/career/online-profiles.rst [hunk 3]
```diff
@@ -14,211 +14,68 @@ things to include, as well as things to leave out of each profile.
...
-You should include a live link to your project, if it is online.
-
-Technical Education
-^^^^^^^^^^^^^^^^^^^
+   Copy document prompt
 
-You need to show how you learned your technical skills. You can do this
-by including college courses you have taken, programs you have been a
-part of, books you have worked through, and MOOC’s you have completed.
+From there, you can pick one of two template formats to edit. Edit only the template text and NOT the formatting. Do NOT delete the LaunchCode brands on the top right. If you already have text from an older resume, you can copy and paste text into your new resume. When doing so, be sure to match destination formatting (Command+SHIFT+V for Macs and CTRL+SHIFT+V for Windows).
```
Comment (CarlyLanglois) [inline src/modules/career/online-profiles.rst]: I would add a note indicating that the second format option is on the second page

File: src/modules/career/online-profiles.rst [hunk 4]
```diff
@@ -236,9 +93,8 @@ handles, gamertags, or anything that is not a variation of your name.
 
 ``a.lovelace`` is a great GitHub profile name for Ada Lovelace!
 
-countessofcomputers is a poor GitHub profile name for Ada Lovelace –
-although she has a steeped background with computers, she should still
-choose a professional github profile name.
+countessofcomputers is a poor GitHub profile name for Ada Lovelace---although she has a steeped background with computers, she should still
```
Comment (CarlyLanglois) [inline src/modules/career/online-profiles.rst]: also countessofcomputers should have syntax highlighting if its the same kind of entity as a.lovelace

File: src/modules/career/online-profiles.rst [hunk 5]
```diff
@@ -14,211 +14,68 @@ things to include, as well as things to leave out of each profile.
...
-    professional communication, you should create one. We personally
-    recommend gmail.
-
-You should also include a link to your GitHub profile, a link to your
-LinkedIn, a physical address, and a link to your personal website (if
-you have one).
-
-Summary of Technical Skills
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For the Apprenticeship Program, we require you to complete a resume using `our template guide <https://docs.google.com/document/d/1Xr0O_ZnNizlWlUsxxCUZHL483NZWV7mmTCbWUhNVm74/view>`_. We require resumes of this format in order to enter our Apprenticeship program. Companies appreciate the shared template and it makes it easier for you to focus on the content of your resume rather than the style
```
Comment (CarlyLanglois) [inline src/modules/career/online-profiles.rst]: I'd change one or both of the mentions of 'apprenticeship program' to have the same capitalization. my preference is both lowercase for these kinds of names but you may have a more formal rule

**Review by CarlyLanglois — APPROVED** [review]
all good! sorry this slipped my mind yesterday. just a few small notes

**Pull request merged**